site-docs/src/components/pricing-battle-k/anderson-styles.css

```
@@ -0,0 +1,302 @@
+/* ═══════════════════════════════════════════════════
+   THE GRAND FEEDTACK HOTEL — Pricing Diorama
+   A Wes Anderson Production
+   ═══════════════════════════════════════════════════ */
+
+@import url("https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,400;0,600;0,700;1,400&family=Josefin+Sans:wght@300;400;600;700&display=swap");
```

Stylelint (17.9.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,400;0,600;0,700;1,400&family=Josefin+Sans:wght@300;400;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Cormorant+Garamond:ital,wght@0,400;0,600;0,700;1,400&family=Josefin+Sans:wght@300;400;600;700&display=swap"" (import-notation)

(import-notation)